apps/src/aichat/views/ChatMessage.tsx

```diff
@@ -103,7 +103,8 @@ const displayModelUpdateMessage = (
         <>
           <span className={moduleStyles.modelUpdateMessageTextContainer}>
             <StrongText>{chatMessageText}</StrongText>
-            {chatMessageSuffix}
+            {chatMessageSuffix?.text}
+            <StrongText>{chatMessageSuffix?.boldtypeText}</StrongText>
```

Comment:
Nit: does this introduce empty elements if there's no boldtypeText? maybe `{chatMessageSuffix?.boldtypeText && <StrongText>...}`